dataland-frontend/tests/component/components/pages/LandingPage.cy.ts

```diff
@@ -0,0 +1,112 @@
+import { checkButton, checkImage, checkAnchorByContent, checkAnchorByTarget } from "@ct/testUtils/ExistenceChecks";
+import NewLandingPage from "@/components/pages/NewLandingPage.vue";
+import { minimalKeycloakMock } from "@ct/testUtils/Keycloak";
+import content from "@/assets/content.json";
+import { type Page, type Section } from "@/types/ContentTypes";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+
+describe("Component test for the landing page", () => {
+  it("Check if essential elements are present", () => {
+    cy.mountWithPlugins(NewLandingPage, {
+      keycloak: minimalKeycloakMock({
+        authenticated: false,
+      }),
+    }).then(() => {
+      validateTopBar();
+      validateIntroSection();
+      validateBrandsSection();
+
```

Comment:
"TheStruggle" is not covered by this test.  What is the reason that it has been left out?